user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -0,0 +1,69 @@
+---
+uid: DashboardGrid
+---
+
+# Grid
+
+> [!IMPORTANT]
+> At present, this component is only available in preview, if the [ReportsAndDashboardsDynamicVisuals](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsdynamicvisuals) soft-launch option is enabled. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+
+Available from DataMiner 10.4.1/10.5.0<!--RN 34761-->. This component allows you to visualize data as a grid.
+
+To configure the component:
+
+1. Apply a data feed. See [Applying a data feed](xref:Apply_Data_Feed).
+
+1. Optionally, hover the mouse over the component, click the filter icon, and then add a filter feed from the *queries* section of the data pane. You can repeat this several times in order to filter on several queries<!--RN 34761-->.
```

Comment:
The only filter that currently makes sense to the grid is the columns from the query filter component, which will be available under Feeds > Query filter >Query Columns